### ISSUE TITLE
Missing GLX thunk defines

### ISSUE THREAD

**Issue by Sonicadvance1** [FEX-Emu/FEX#1894]
```
template<> struct fex_gen_config<glXWaitX> {};
template<> struct fex_gen_config<glXGetGPUIDsAMD> {};
template<> struct fex_gen_config<glXGetGPUInfoAMD> {};
template<> struct fex_gen_config<glXGetContextGPUIDAMD> {};
template<> struct fex_gen_config<glXCreateAssociatedContextAMD> {};
template<> struct fex_gen_config<glXCreateAssociatedContextAttribsAMD> {};
template<> struct fex_gen_config<glXDeleteAssociatedContextAMD> {};
template<> struct fex_gen_config<glXMakeAssociatedContextCurrentAMD> {};
template<> struct fex_gen_config<glXGetCurrentAssociatedContextAMD> {};
template<> struct fex_gen_config<glXBlitContextFramebufferAMD> {};
template<> struct fex_gen_config<glXGetCurrentDisplayEXT> {};
template<> struct fex_gen_config<glXGetAGPOffsetMESA> {};
template<> struct fex_gen_config<glXCreateGLXPixmapMESA> {};
template<> struct fex_gen_config<glXReleaseBuffersMESA> {};
template<> struct fex_gen_config<glXSet3DfxModeMESA> {};
template<> struct fex_gen_config<glXCopyBufferSubDataNV> {};
template<> struct fex_gen_config<glXNamedCopyBufferSubDataNV> {};
template<> struct fex_gen_config<glXCopyImageSubDataNV> {};
template<> struct fex_gen_config<glXDelayBeforeSwapNV> {};
template<> struct fex_gen_config<glXEnumerateVideoDevicesNV> {};
template<> struct fex_gen_config<glXBindVideoDeviceNV> {};
template<> struct fex_gen_config<glXJoinSwapGroupNV> {};
template<> struct fex_gen_config<glXBindSwapBarrierNV> {};
template<> struct fex_gen_config<glXQuerySwapGroupNV> {};
template<> struct fex_gen_config<glXQueryMaxSwapGroupsNV> {};
template<> struct fex_gen_config<glXQueryFrameCountNV> {};
template<> struct fex_gen_config<glXResetFrameCountNV> {};
template<> struct fex_gen_config<glXBindVideoCaptureDeviceNV> {};
template<> struct fex_gen_config<glXEnumerateVideoCaptureDevicesNV> {};
template<> struct fex_gen_config<glXLockVideoCaptureDeviceNV> {};
template<> struct fex_gen_config<glXQueryVideoCaptureDeviceNV> {};
template<> struct fex_gen_config<glXReleaseVideoCaptureDeviceNV> {};
template<> struct fex_gen_config<glXGetVideoDeviceNV> {};
template<> struct fex_gen_config<glXReleaseVideoDeviceNV> {};
template<> struct fex_gen_config<glXBindVideoImageNV> {};
template<> struct fex_gen_config<glXReleaseVideoImageNV> {};
template<> struct fex_gen_config<glXSendPbufferToVideoNV> {};
template<> struct fex_gen_config<glXGetVideoInfoNV> {};
template<> struct fex_gen_config<glXQueryHyperpipeNetworkSGIX> {};
template<> struct fex_gen_config<glXHyperpipeConfigSGIX> {};
template<> struct fex_gen_config<glXQueryHyperpipeConfigSGIX> {};
template<> struct fex_gen_config<glXDestroyHyperpipeConfigSGIX> {};
template<> struct fex_gen_config<glXBindHyperpipeSGIX> {};
template<> struct fex_gen_config<glXQueryHyperpipeBestAttribSGIX> {};
template<> struct fex_gen_config<glXHyperpipeAttribSGIX> {};
template<> struct fex_gen_config<glXQueryHyperpipeAttribSGIX> {};
template<> struct fex_gen_config<glXBindSwapBarrierSGIX> {};
template<> struct fex_gen_config<glXQueryMaxSwapBarriersSGIX> {};
template<> struct fex_gen_config<glXJoinSwapGroupSGIX> {};
template<> struct fex_gen_config<glXBindChannelToWindowSGIX> {};
template<> struct fex_gen_config<glXChannelRectSGIX> {};
template<> struct fex_gen_config<glXQueryChannelRectSGIX> {};
template<> struct fex_gen_config<glXQueryChannelDeltasSGIX> {};
template<> struct fex_gen_config<glXChannelRectSyncSGIX> {};
template<> struct fex_gen_config<glXCushionSGI> {};
template<> struct fex_gen_config<glXGetTransparentIndexSUN> {};
template<> struct fex_gen_config<glXBindTexImageARB> {};
template<> struct fex_gen_config<glXReleaseTexImageARB> {};
template<> struct fex_gen_config<glXDrawableAttribARB> {};
template<> struct fex_gen_config<glXGetFrameUsageMESA> {};
template<> struct fex_gen_config<glXBeginFrameTrackingMESA> {};
template<> struct fex_gen_config<glXEndFrameTrackingMESA> {};
template<> struct fex_gen_config<glXQueryFrameTrackingMESA> {};
```

A fair number of these missing from glxext.h

**Comment by Sonicadvance1** [comment]
Looks like all of these are simple to implement. The only one throwing errors is `glXCopyImageSubDataNV` but that is just because it has more arguments than we currently handle

**Comment by Sonicadvance1** [comment]
Fixed with #1897
